GLTFSDK.Test/Source/AnimationUtilsTests.cpp

```diff
@@ -107,75 +107,90 @@ namespace Microsoft
                     bufferBuilder.AddBufferView(BufferViewTarget::ARRAY_BUFFER);
 
                     std::vector<float> input = {
-                        0.213941514f, 0.963860869f, -0.158749819f, 0.204712942f
+                        0.213941514f, 0.963860869f, 0.204712942f
                     };
-                    auto accessor = bufferBuilder.AddAccessor(input, { TYPE_VEC4, COMPONENT_FLOAT });
+                    auto accessor = bufferBuilder.AddAccessor(input, { TYPE_VEC3, COMPONENT_FLOAT });
 
                     Document doc;
                     bufferBuilder.Output(doc);
 
                     // Accessor
 
                     GLTFResourceReader reader(readerWriter);
-                    auto output = AnimationUtils::GetRotations(doc, reader, accessor);
+                    auto output = AnimationUtils::GetScales(doc, reader, accessor);
 
                     AreEqual(input, output);
 
                     // Sampler
 
                     AnimationSampler animationSampler;
                     animationSampler.outputAccessorId = accessor.id;
-                    output = AnimationUtils::GetRotations(doc, reader, animationSampler);
+                    output = AnimationUtils::GetScales(doc, reader, animationSampler);
 
                     AreEqual(input, output);
                 }
 
-                GLTFSDK_TEST_METHOD(AnimationUtilsTests, AnimationUtils_Test_GetScales_Vec3_Float)
+                // Utility for verifying GetMorphWeights
+                static std::map< std::type_index, ComponentType> kComponentTypeMap;
```

Comment:
If you're going to stick with this map, rather than use a second template parameter, then it should be const

Reply:
Also, it's not a class member so no need for static

Reply:
Interestingly, const-ing the map broke the code that uses std::type_index as a map key for operator [], since there's no default value for a std::type_index. That's now fixed.

Somewhat confusingly, the map IS a class member - but only under the CppUnitTest framework used in the VS solution. With GoogleTest used by the CMake build and GitHub, it isn't. 
Latest code works for both.